lib/workos/webhooks.rb

```diff
@@ -86,7 +86,9 @@ def verify_header(
           )
         end
 
-        if timestamp < Time.now - tolerance
+        timestamp_to_time = Time.at(timestamp.to_i / 1000)
```

Comment:
I confirmed that the timestamp we send is in milliseconds, not seconds, and so this fix makes sense. 👍